build

```diff
@@ -8,10 +9,13 @@ try {
 	process.exit(1);
 }
 
-var child_process = require('child_process');
-var fs = require('fs');
-var path = require('path');
-var force = process.argv[2] === '--force';
```

Comment:
IIRC these are intended to be `var`, not `const`, so that really old node versions will reach a point where a more descriptive error can be thrown.

Reply:
I don't really think that decision still makes sense several years later, but that's a separate change.

Reply:
Yeah, we should probably put the explanation in comments at the top of the file, so this doesn't keep happening.

Reply:
Noted.

Reply:
I reverted any eslint auto-fixed changes and added exceptions to `.eslintrc-no-types.json`.